act/qc/qc_summary.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
```

Comment:
Can we add a general docstring summary above the imports for this new module

Reply:
Agree here

Reply:
Done.